### PR TITLE
874256-UG documentation added for .NET MAUI Theme support

### PR DESCRIPTION
**Pull request by AarthiArjunan**
### **Description**

- Added UG content for the .NET MAUI theme support

#### **Added topics**

- Applying themes

- How to

### CODE REVIEW

File: maui-toc.html [hunk 1]
```diff
@@ -47,19 +47,19 @@
                     </li>
                 </ul>
             </li>
-			<li>
-				<a href="/maui/installation/install-nuget-packages">Install through NuGet Packages</a>
-			</li>
+            <li>
```
Comment (naveenkumar-sanjeevirayan) [inline maui-toc.html:50]: rever the unwanted changes

File: MAUI/ListView/Pull-To-Refresh.md [hunk 2]
```diff
@@ -1,52 +1,24 @@
 ---
 layout: post
 title: Pull To Refresh in .NET MAUI ListView control | Syncfusion
-description: Learn here all about Pull To Refresh support in Syncfusion .NET MAUI ListView (SfListView) control and more.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/ListView/Pull-To-Refresh.md:4]: why the unnecessary changes are showing, can you remove it.

File: MAUI/Themes/HowTo.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    ICollection<ResourceDictionary> mergedDictionaries = Application.Current.Resources.MergedDictionaries;
+    if (mergedDictionaries != null)
+    {
+        var theme = mergedDictionaries.OfType<SyncfusionThemeResourceDictionary>().FirstOrDefault();
+        if (theme != null)
+        {
+            if (theme.SfVisualTheme is SfVisuals.DarkDefault)
+            {
+                theme.SfVisualTheme = SfVisuals.LightDefault;
+                Application.Current.UserAppTheme = AppTheme.Light;
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:26]: No need to add the application theme code changes

File: MAUI/Themes/HowTo.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                theme.SfVisualTheme = SfVisuals.DarkDefault;
+                Application.Current.UserAppTheme = AppTheme.Dark;
+            }
+        }
+     }
+}
+
+{% endhighlight %}
+
+Similarly, to switch to dark theme from light theme, remove the light theme resource, and add the dark theme resource dictionary.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:39]: In the above code snippet, we explained it. So, remove it.

File: MAUI/Themes/HowTo.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
+To switch to light theme from dark theme, use the following code snippet.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:12]: To switch between light and dark themes in Syncfusion Maui controls, use the following code snippet.

File: MAUI/Themes/Themes.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+* Applying light and dark themes
+* Overriding the default theme
+* Creating your own theme
+
+## Applying light and dark themes
+
+By default, Syncfusion provides support for light and dark themes. As names suggest, these themes will have colors with light and dark color contrasts, respectively.
+
+To apply themes to your application, merge the following items:
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:22]: To apply themes to your application, merge the SyncfusionThemeResourceDictionary.

File: MAUI/Themes/Themes.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+* Theme resource dictionary
+
+#### Theme resource dictionary
+
+This resource dictionary contains *keys* and their respective color codes for all the Syncfusion controls. Currently, the following two theme resource dictionaries are available:
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/Themes.md:28]: Explain this section properly. say about SfVisual property.

File: MAUI/Themes/HowTo.md (hunk 3 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:26]: Changes addressed.

File: MAUI/Themes/HowTo.md (hunk 4 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:39]: Changes addressed.

File: MAUI/Themes/HowTo.md (hunk 5 above)
Comment (AarthiArjunan) [reply MAUI/Themes/HowTo.md:12]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 6 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:22]: Changes addressed.

File: MAUI/Themes/Themes.md (hunk 7 above)
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:28]: Changes Addressed.

File: MAUI/Themes/Themes.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+<Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary SfVisualTheme="DarkDefault"/>
+            <ResourceDictionary>
+                <Color x:Key="SfRadialGaugeNeedlePointerNeedleFillColor">LimeGreen</Color>
```
[the remark this answers is not in the document]
Comment (AarthiArjunan) [reply MAUI/Themes/Themes.md:79]: Yes

File: maui-toc.html (hunk 1 above)
Comment (AarthiArjunan) [reply maui-toc.html]: Changes addressed

File: MAUI/Themes/HowTo.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Themes/HowTo.md:11]: remove the unwanted empty lines
